addon.py

```
@@ -139,38 +182,68 @@ def _handle_client(self, client):
+                            if decoded is not None:
+                                is_incomplete = (
+                                    exc.pos == len(decoded)
+                                    or exc.msg.startswith("Unterminated string")
+                                    or exc.msg.startswith("Expecting value")
+                                )
+                                if is_incomplete:
+                                    # Incomplete JSON, wait for more data
+                                    break
+                                
+                                # Malformed JSON
+                                print(f"Malformed JSON received: {str(exc)}")
```

Ruff (0.15.6)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)